core/commands/generate_repository_partials/clustrix_parser.rb

```diff
@@ -50,6 +51,14 @@ def self.get_release_paths(path_uri, paths, link_name)
     end.flatten
   end
 
+  def self.determine_product_name(link)
```

Comment:
Make it a parameter of the parser. It should be configured outside of the parser.